### PR TITLE
Add NV LTC scraper

### PR DESCRIPTION
**Pull request by zachlipton**

### CODE REVIEW

File: data-collection-scripts/ltc-scraper/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (julia326) [inline data-collection-scripts/ltc-scraper/LICENSE]: why its own here vs the repo-wide license? (Ditto on the .gitignore contents, could add to the repo-wide one)

File: data/ltc_nv.csv [hunk 2]
```diff
@@ -0,0 +1,178 @@
+Facility Name,Facility Type,County,Unique Confirmed Cases,Recovered Residents/Patients,Recovered Staff,Deaths Residents/Patients,Deaths Staff,Attack Rate Residents/Patients,Attack Rate Staff,Confirmed COVID-19 Cases Residents/Patients,Confirmed COVID-19 Cases Staff,Confirmed COVID-19 Cases Imported
```
Comment (julia326) [inline data/ltc_nv.csv:1]: I think this is a copy of the `nv_ltc.csv` above?

**Comment by zachlipton** [comment]
Ah good catch. A few things were leftover from the repository move. 

I deleted the separate license, consolidated .gitignore, and removed the extra data directory (the GH Actions job will write to the repo-level data directory)

File: .github/workflows/update_ltc.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+            restore-keys: |
+              ${{ runner.os }}-node-
+        - name: Use Node.js
+          uses: actions/setup-node@v1
+          with:
+            node-version: '14.x'
+        - name: Install dependencies
+          run: npm install
+        # there's an extremely irritating DNS bug somewhere (https://github.com/actions/virtual-environments/issues/798)
```
Comment (julia326) [inline .github/workflows/update_ltc.yml:29]: hahah wow

File: data-collection-scripts/ltc-scraper/.gitignore [hunk 4]
```diff
@@ -0,0 +1,106 @@
+# Logs
```
Comment (julia326) [inline data-collection-scripts/ltc-scraper/.gitignore]: this one's still here?

**Review by julia326 — APPROVED** [review]
Good to merge once you remove the extra .gitignore file :)

File: .github/workflows/update_ltc.yml (hunk 3 above)
Comment (zachlipton) [reply .github/workflows/update_ltc.yml:29]: It’s not DNS
There is a no way it’s DNS
It was DNS

Do not get me started. It was extraordinarily annoying

File: data-collection-scripts/ltc-scraper/.gitignore (hunk 4 above)
Comment (zachlipton) [reply data-collection-scripts/ltc-scraper/.gitignore]: Just a sec. Fighting with the GH Actions commit action, but I'll make sure it's gone

**Pull request merged**